.github/workflows/ci.yml

```diff
@@ -8,31 +8,27 @@ on:
 
 jobs:
   pod-lint:
-    name: Pod Lint
+    name: CocoaPods Lint
     runs-on: macos-latest
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v4
-      - name: Bundle Install
-        run: bundle install --gemfile=Example/Gemfile
-      - name: Pod Install
-        run: bundle exec --gemfile=Example/Gemfile pod install --project-directory=Example
-      - name: Lint Stagehand Podspec
-        run: bundle exec --gemfile=Example/Gemfile pod lib lint --verbose --fail-fast Stagehand.podspec
-      - name: Lint StagehandTesting Podspec
-        run: bundle exec --gemfile=Example/Gemfile pod lib lint --verbose --fail-fast --include-podspecs=Stagehand.podspec StagehandTesting.podspec
+      - name: Lint podspecs
+        run: pod lib lint --verbose --fail-fast --include-podspecs=Stagehand.podspec StagehandTesting.podspec
```

Comment:
I had removed that because the root of the repo doesnt have a `Gemfile` and it was referring to `Example` here which i was going to address build & testing in #77. I added back the `bundle exec` until we add a root `Gemfile`